1_basics.ipynb

```diff
@@ -519,8 +519,8 @@
    },
    "source": [
     "<div class=\"alert alert-block alert-info\">\n",
-    "    <b>What's next</b>\n",
-    "    <p>In the next notebook, we will capture the information from the sensors mounted on HSR, and move the mobile base until it reach to the front of the wall.</p>\n",
+    "    <b>What's next?</b>\n",
+    "     <p>In the next notebook, we will capture the information from the sensors mounted on **the** HSR, and move the mobile base until it reaches the front of the wall.</p>\n",
```

Comment:
Ah.  The "**the**" here needs to have the "**" removed from around it.
It appears that you have accidentally included the bold font that I used to try to highlight what needed to change for this sentence.